content/working_with/console/widgets/secretProviders.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Secret Providers Management
+category: Widgets
+draft: false
+---
+
+Displays all the secret providers visible to the logged-in user in the current tenant (including global secrets). 
```

Comment:
What does "(including global secrets)" mean? 🤔

Reply:
I based my documentation mainly on secrets widget https://docs.cloudify.co/latest/working_with/console/widgets/secrets/ but I guess I copied too much this time 😅 adjusted: https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2498/commits/18bfea37a46cfa875ff053049ef8580e7be1a7fe

Reply:
So are we going to keep "(including global secrets)"?

Reply:
https://github.com/cloudify-cosmo/docs.getcloudify.org/pull/2498/commits/b908e20bef3fe9922f2ad05efc655306ab170e33 no, no, removed ✔️